### DISCUSSION TITLE
Giving admin permission to a collaborator<title>

### DISCUSSION THREAD

**Discussion by Sari727896** [community/community#131415] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

How can I give administrator permission to one of the repository participants?

**Comment by davevad93** [comment]
Hi @Sari727896 ,

A repo owned by a personal account has two permission levels: the owner and collaborators. For personal accounts, to give permissions in a repo to a contributor you need to add him/her as a `Collaborator`, to do this go to the repo `Settings` => `Collaborators` and invite the user/s you want to give access to, once they accept your invitation, they will have collaborator access to your repo. 

![0](https://github.com/community/community/assets/94308234/560103ea-57e7-415f-9583-34568c3face8)

If you need to give a contributor more granular access, for example by setting different levels of permissions, you need an organization account.
